xcp_d/utils/plot.py

```diff
@@ -512,9 +512,12 @@ def plot_svgx(rawdata,
 
     # Remove first N deleted from raw_data so it's same length as censored files
     if len(raw_dvars_data) > len(residual_dvars_data):
+        # TODO: Should this be [-len(residual_dvars_data):] ?
+        # ... Seems to grab first N, not last N.
         raw_dvars_data = raw_dvars_data[0:len(residual_dvars_data)]
         raw_data = raw_data[:, 0:len(residual_dvars_data)]
-        # regressed_dvars_data = raw_dvars_data #TODO: Check if this is needed
+        # TODO: Figure out how to slice the regressed DVARS instead of just overwriting it
+        regressed_dvars_data = raw_dvars_data
```

Comment:
@kahinimehta it looks like you added the comment in #447. Following up here, since the mismatch in length between `raw_dvars_data` and `regressed_dvars_data` was raising an error in this PR a few lines down where the DataFrame is created.

I don't know the full set of conditions that cause a problem, but at least one of them is having >0 volumes censored. I think the way the cropping is done here is wrong, and I have added a TODO comment to that effect. I also think it's weird to overwrite the regressed DVARS with the raw DVARS, but I'm happy to keep the hack in for now, since it's outside the scope of this PR.

Reply:
I've punted the issue to #589. Hopefully we can find a solution, but I don't want to address it in this PR.